### DISCUSSION TITLE
Can I skip Face detection (I have previously detected) and go straight to getting embeddings?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [exadel-inc/CompreFace#877] [Q&A] [answered]
Can I skip Face detection (I have previously detected) and go straight to getting embeddings?  
I encountered this issue - that mostly on group photos get this error:
https://github.com/exadel-inc/CompreFace/issues/874

detection_with_embedings: DetectionService = compre_face.init_face_detection(DETECTION_API_KEY)
compre_face_only_boxes: CompreFace = CompreFace(DOMAIN, PORT)
detection_only_boxes: DetectionService = compre_face_only_boxes.init_face_detection(DETECTION_API_KEY)

But detection only works fine for these.

Is it possible to detect bounding boxes of faces, then crop them and then go straight to getting embeddings, without face detection phase?

**Comment by pospielov** [comment, marked as answer]
It's not possible with the current build
I'll create a task, and we will review it.
I think it's possible to skip a detection step if we cancel the execution of all other plugins except embeddings, as they may depend on the detection step.
